src/lib/common/defaultValues.h

```diff
@@ -28,8 +28,15 @@
 
 
 
+/* ****************************************************************************
+*
+* Default Service Paths
+*/
 #define  DEFAULT_SERVICE_PATH_UPDATES         "/"
 #define  DEFAULT_SERVICE_PATH_QUERIES         "/#"
+#define  DEFAULT_SERVICE_PATH_REGISTRATIONS   "/"
```

Comment:
I'd suggest not adding a new one and usings DEFAULT_SERVICE_PATH_UPDATES. These constants have to be interpreted in general case. I mean, UPDATE doesn't mean "NGSI10 update" but any operation with updating semantics (i.e. changing things) procesed by the CB, either in the NGSI10 or NGSI9 cases.

Reply:
In fact, registrations should use `/#`. By default, a registrations forwards request in any service path, so `/#` is the one to use.

Reply:
On a second though to this maybe the DEFAULT_SERVICE_PATH_UPDATES and DEFAULT_SERVICE_PATH_QUERY names for the constans are confusing, as the update/query semantics breaks in the case of registration/subscriptions.

Maybe is better to use:

```
 #define  SERVICE_PATH_ROOT         "/"
 #define  SERVICE_PATH_ALL          "/#"
```

Reply:
Sounds good to me, however, I still believe that using /# for registrations is a bad idea.

Reply:
Fixed in 1a49367

Reply:
From https://fiware-orion.readthedocs.io/en/master/user/service_path/index.html:

> While entities belong to services and servicepaths, subscriptions and registrations belong only to the service. The servicepath in subscriptions and registrations doesn't denote sense of belonging, but is the expression of the query associated to the subscription or registration.

Under that ligth, I think is a good idea that the query associated be as wider as possible in the default case, thus `/#` is the one to use.

Reply:
So, let's agree to disagree.
Discussion needed.
Creating an issue